tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -530,6 +530,9 @@ void WriteDictionary(IReadOnlyDictionary<string, string> dictionary)
                     writer.WritePropertyName("service_mapping");
                     WriteDictionary(instanceSettings.ServiceNameMappings);
 
+                    writer.WritePropertyName("dbm_propagation_mode");
+                    writer.WriteValue(Environment.GetEnvironmentVariable("DD_DBM_PROPAGATION_MODE"));
```

Comment:
The only thing with this is that it would give me the enum values, 0, 1, 2, so I was trying to get the actual string value but didn't now how to, will look into that, thank you!

Reply:
> The only thing with this is that it would give me the enum values, 0, 1, 2, so I was trying to get the actual string value

If you call `ToString()` on an `enum` value, it returns the `string` representation of it, so this should work:

```suggestion
                    writer.WriteValue(instanceSettings.DbmPropagationMode.ToString());
```

Be aware that this calling `ToString()` on an enum uses reflection and can be slow, so we generally avoid it where possible (I have [a post about it here](https://andrewlock.net/netescapades-enumgenerators-a-source-generator-for-enum-performance/), and why we use source generators! 🙂), but given that this is happening in the background, I don't think it's a problem in this case